src/headless_browser.ts

```diff
@@ -15,7 +17,29 @@
 //     throw new Error("Unhandled result type: " + result["result"]["type"])
 // }
 
-import { readLines } from "../deps.ts";
+interface MessageResponse { // For when we send an event to get one back, eg running a JS expression
+  id: number;
+  result?: {
+    result: { [key: string]: unknown };
```

Comment:
FYI; not all responses have an "inner" result, its common but not always.

Reply:
Ok cheers, gotcha 👍 thanks !

Reply:
https://github.com/drashland/dawn/pull/19/commits/44dbe7f41fdf810d8cdc1fcf81542b17d6fc2eab